go.mod

```diff
@@ -24,3 +24,5 @@ require (
 retract v1.0.22 // old gx tag accidentally pushed as go tag
 
 retract v2.0.1+incompatible // old gx tag
+
+retract v2.0.2-retract // we need to use a newer version than v2.0.1 to retract v2.0.1+incompatible, but we can retract ourself directly once done
```

Comment:
CI is failing with: `go.mod:28: retract github.com/ipfs/go-ipfs-cmds: version "v2.0.2-retract" invalid: should be v0 or v1, not v2`

It looks like this might be because we'd need to change the import path in this go mod file as well https://github.com/golang/go/issues/35732#issuecomment-584319096. I haven't dealt with two many v2 modules like this before though so lmk if I'm missing something.

Reply:
Side note: Does this need to merge into master or can this merge into a `v2.0.2-retract-branch` and be released from there?